src/main.cpp

```diff
@@ -87,8 +87,8 @@ BuzzerController buzzer(BUZZER_ON_INTERVAL);
 
 SafetySystem safety_system(&ams_interface, &wd_interface);
 // SafetySystem safety_system(&ams_interface, &wd_interface, &dashboard);
-PedalsSystem pedals_system({ACCEL1_PEDAL_MIN, ACCEL2_PEDAL_MIN, ACCEL1_PEDAL_MAX, ACCEL2_PEDAL_MAX, APPS_ACTIVATION_PERCENTAGE, DEFAULT_PEDAL_DEADZONE, DEFAULT_PEDAL_IMPLAUSIBILITY_MARGIN, APPS_ACTIVATION_PERCENTAGE},
-                           {BRAKE1_PEDAL_MIN, BRAKE2_PEDAL_MIN, BRAKE1_PEDAL_MAX, BRAKE2_PEDAL_MAX, BRAKE_ACTIVATION_PERCENTAGE, DEFAULT_PEDAL_DEADZONE, DEFAULT_PEDAL_IMPLAUSIBILITY_MARGIN, BRAKE_MECH_THRESH});
+PedalsSystem pedals_system({ACCEL1_PEDAL_MIN, ACCEL2_PEDAL_MIN, ACCEL1_PEDAL_MAX, ACCEL2_PEDAL_MAX, ACCEL1_PEDAL_OOR_MIN, ACCEL2_PEDAL_OOR_MIN, ACCEL1_PEDAL_OOR_MAX, ACCEL2_PEDAL_OOR_MAX, APPS_ACTIVATION_PERCENTAGE, DEFAULT_PEDAL_DEADZONE, DEFAULT_PEDAL_IMPLAUSIBILITY_MARGIN, APPS_ACTIVATION_PERCENTAGE},
```

Comment:
Stylistically, passing all of these parameters like this is very hard to follow, since the OOR values are static and will not be changing, these can just be defined and used in the pedals system rather than being passed in through pedalparams